src/test/StateController.test.js

```diff
@@ -0,0 +1,34 @@
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { StateController } from '../main/StateController';
+
+describe('State Controller', () => {
+    test('render individual states', () => {
+        //test neutral state
+        render(<StateController initState={"Neutral"}/>);
+        screen.debug();
+        expect(screen.getByRole("Neutral"));
```

Comment:
I don't think this will work because react components don't render their own tags in the final html, see the debug printout above. You will need to try to get something that is unique to the neutral state. If there is nothing, then that says something about your html, but you can just add an ID to it for testing, see the test framework documentation. You will also need a .toBe or some other method call after the expect... You need to say what you're expecting.